backtracepython/request_handler.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+import requests
+import simplejson as json
+
+
+class BacktraceRequestHandler:
+    def __init__(self, submission_url, timeout, ignore_ssl_certificate, debug):
+        self.submission_url = submission_url
+        self.timeout = timeout
+        self.ignore_ssl_certificate = ignore_ssl_certificate
+        self.debug = debug
+
+    def send(self, report, attachments):
+        payload = json.dumps(report, ignore_nan=True, bigint_as_string=True)
+        self.debug_api(
+            "Submitting a payload to {},\n {}\n".format(self.submission_url, payload)
+        )
```

Comment:
maybe it would be a better idea to format in the `debug_api` method? This way you won't have to do string formatting if debugging is disabled

Reply:
On my todo list - thanks for pointing out!